pkg/adsc/delta.go

```diff
@@ -262,6 +267,11 @@ func (c *Client) Run(ctx context.Context) {
 	c.runWithReconnects(ctx)
 }
 
+// Synced returns a channel that will close once the client has received all initial watches
+func (c *Client) Synced() <-chan struct{} {
```

Comment:
Provide HasSynced, then no need synced chan

Reply:
A synced channel is strictly better than has synced. HasSynced requires you to poll, a channel allows you to poll OR wait until its ready

Reply:
Wondering should align with Informer or not. You are right, this is easier to use.